### PR TITLE
add getScreenMedia support for Chrome

### PR DESCRIPTION
**Pull request by fippo**
tentative support, Chrome needs to ship first:  https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/j7k2nI_9nng

### CODE REVIEW

File: getscreenmedia.js [hunk 1]
```diff
@@ -6,11 +6,17 @@ module.exports = function (constraints, cb) {
...
-        // check that the extension is installed by looking for a
-        // sessionStorage variable that contains the extension id
-        // this has to be set after installation unless the content
-        // script does that
-        if (sessionStorage.getScreenMediaJSExtensionId) {
+        if ('getDisplayMedia' in window.navigator) { // prefer spec getDisplayMedia
+            window.navigator.getDisplayMedia({video: true}).then(function (stream) {
+                callback(null, stream);
+            }).catch(function (err) {
+                callback(err);
```
Comment (xdumaine) [inline getscreenmedia.js]: We should move this to promise based as well.

**Review by xdumaine — APPROVED** [review]

File: getscreenmedia.js (hunk 1 above)
Comment (fippo) [reply getscreenmedia.js]: I actually hope to retire this module soon (TM)

**Comment by xdumaine** [comment]
+1 for removing webrtc-adapter as a dep here. This is one of the reasons we end up getting it pulled in multiple times.

**Comment by fippo** [comment]
well, in talky i simply removed its dependency on getScreenMedia :-)

**Comment by xdumaine** [comment]
Sadly, until I get time to refactor more things, I still need this :(

**Comment by fippo** [comment]
but now it won't work with the adapter version that only does navigator.mediaDevices.getUserMedia?

(looks like I forgot about this :-|)

**Pull request merged**